### PR TITLE
Added the ability read the notes from ISIS nexus files

### LINKED ISSUE #34908: Muon v2 data reader missing comment
From Steve.

While updating our v2 data definition document I happened to notice that the run comment isn't displayed when loading a v2 file - see pictures below. The text needed seems to be present just fine in the v2 file at 'raw_data_1/notes'.

Loading a 'v1' file ... note that text follows 'Comment':
image

Loading a 'v2' file ... note lack of text following 'Comment':
image

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
The muon group wanted to be able to read the notes from a nxs file. Turns out that it a field in all ISIS nxs files. I have updated the main reader to handle this. 

**To test:**

Load files from different instruments. The below script is a nice way to check things (EMU and IRIS have comments)

```
files = ['EMU115363.nxs_v2', 'POLREF00004699.nxs', 'IRIS56944.nxs']

for file in files:
    ws = LoadISISNexus(file)
    if "POL" in file:
        # returns a group
        ws = ws[0]
    print("file: ", file)
    print("comment: ", ws.getComment())
    print()
```

Fixes #34908 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/DataHandling/test/LoadISISNexusTest.h [hunk 1]
```diff
@@ -1458,6 +1458,36 @@ class LoadISISNexusTest : public CxxTest::TestSuite {
                       "select monitors, but to also exclude them",
                       ld.execute(), const std::invalid_argument &);
   }
+
+  void testReadNotes() {
+
+    LoadISISNexus2 load;
+    load.initialize();
+    load.setPropertyValue("Filename", "EMU000102347.nxs_v2");
```
Comment (AnthonyLim23) [inline Framework/DataHandling/test/LoadISISNexusTest.h:1466]:
```suggestion
    load.setPropertyValue("Filename", "EMU00102347.nxs_v2");
```

**Comment by gemmaguest** [comment]
Note to whoever merges this that it would be good to do a squash and merge because we will need to cherry-pick the fix onto another branch.

**Review by cailafinn — CHANGES_REQUESTED** [review]
Works well, just a quick comment on the release notes

**Review by cailafinn — APPROVED** [review]
Looks good to me now!

**Pull request merged**